.github/workflows/deploy-enterprise.yml

```diff
@@ -18,13 +18,12 @@ jobs:
             domain: hyp3-its-live.asf.alaska.edu
             template_bucket: cf-templates-3o5lnspmwmzg-us-west-2
             image_tag: latest
-            product_lifetime_in_days: 45
+            product_lifetime_in_days: 14
```

Comment:
How that we automagically publish products to the ITS_LIVE AWS Open Data bucket as the last step in AUTORIFT jobs, we don't need to retain products long term in this deployment